### PR TITLE
upgrade-to-2.11.x

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR:

- upgrades keda chart using vendir to the 2.11.2 app version so this supports from kubernetes 1.25 to 1.27

### Testing

Description on how keda-app can be tested.

- [ ] fresh install works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM
- [ ] upgrade from previous version works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM

#### Other testing

Description of features to additionally test for keda-app installations.

- [ ] check reconciliation of existing resources after upgrading
- [ ] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Review by Gacko — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Thanks @Gacko I'm waiting for customer testing before I merge this :)

**Comment by Gacko** [comment]
No, thank you for doing that job! 🎉

**Pull request merged**
